app/src/main/res/values/arrays.xml

```diff
@@ -63,5 +63,48 @@
         <item>( ͡° ͜ʖ ͡°)</item>
         <item>( ͡° ʖ̯ ͡°)</item>
         <item>( ͡º ͜ʖ͡º)</item>
+        <item>( ͡°( ͡° ͜ʖ( ͡° ͜ʖ ͡°)ʖ ͡°) ͡°)</item>
+        <item>(⌐ ͡■ ͜ʖ ͡■)</item>
+        <item>(╥﹏╥)</item>
+        <item>(╯︵╰,)</item>
+        <item>(ʘ‿ʘ)</item>
+        <item>(｡◕‿‿◕｡)</item>
+        <item>(⇀‸↼‶)</item>
+        <item>(òóˇ)</item>
+        <item>(✌ ﾟ ∀ ﾟ)☞</item>
+        <item>ʕ•ᴥ•ʔ</item>
+        <item>ᶘᵒᴥᵒᶅ</item>
+        <item>(⌒(oo)⌒)</item>
+        <item>ᄽὁȍ ̪ őὀᄿ</item>
+        <item>( ͡€ ͜ʖ ͡€)</item>
+        <item>( ͡° ͜ʖ ͡°)</item>
+        <item>( ͡° ͜ʖ ͡°)ﾉ⌐■-■</item>
+        <item>(⌐ ͡■ ͜ʖ ͡■)</item>
+        <item>¯\_(ツ)_/¯</item>
+        <item>(ꖘ⏏ꖘ)</item>
+        <item>(╯°□°）╯︵ ┻━┻</item>
+        <item>( ͡~ ͜ʖ ͡°)</item>
+        <item>( ಠ_ಠ)</item>
+        <item>(・へ・)</item>
+        <item>(ง✿﹏✿)ง</item>
+        <item>(づ•﹏•)づ</item>
+        <item>乁(♥ ʖ̯♥)ㄏ</item>
+        <item>|૦ઁ෴૦ઁ|</item>
+        <item>乁(⫑ᴥ⫒)ㄏ</item>
+        <item>(ꖘ‸ꖘ)</item>
+        <item>(✿ ͟ʖ✿)</item>
+        <item>(งⱺ ͟ل͜ⱺ)ง</item>
+        <item>(￣෴￣)</item>
+        <item>ヽ( ͠°෴ °)ﾉ</item>
+        <item>└[⚆ᴥ⚆]┘</item>
+        <item>ヽ(☼ᨓ☼)ﾉ</item>
+        <item>XD</item>
```

Comment:
XD

Reply:
no ej, sam sie zdziwilem ze tam jest xD (bo to kopiuj wklej z portalu)